.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+/package-lock.json
```

Comment:
Great, that you added this. If you could also add, .idea file here and delete one from the project would appreciate it.

Reply:
what do you mean by "delete one from the project"?

Reply:
In your [project](https://github.com/YogiHa/visual-music-1/tree/contact-form) you have .idea folder. So in .gitignore add .idea file to prevent it from being committed. As well as delete .idea folder from your project and commit.